src/app/component/ProductDetails/ProductDetails.component.js

```diff
@@ -25,15 +25,15 @@ class ProductDetails extends Component {
      * Render product SKU only when it's loaded
      */
     renderSku() {
-        const { product: { variants }, areDetailsLoaded, configurableVariantIndex } = this.props;
+        const { product: { variants, sku }, areDetailsLoaded, configurableVariantIndex } = this.props;
 
-        if (variants && variants[configurableVariantIndex] && areDetailsLoaded) {
-            const { product } = variants[configurableVariantIndex];
+        if (areDetailsLoaded) {
+            const { product } = variants && configurableVariantIndex ? variants[configurableVariantIndex] : '';
```

Comment:
You are going to destruct `''`?

Reply:
this destructs variants[configurableVariantIndex], and if it doesn't exists product becomes ''